ee/tabby-webserver/src/lib.rs

```diff
@@ -40,14 +42,32 @@ pub async fn attach_webserver(
     let ctx = Arc::new(ServerContext::new(conn, logger, code));
     let schema = Arc::new(create_schema());
 
+    repositories::load_dataset().await.unwrap();
```

Comment:
Extract into a function

Reply:
I didn't get your point, `load_dataset` is already one single function call, do you mean to put it in another function then log the error?

Reply:
Hey - i mean that changes between line 45 to 61 can be extracted into a function?